src/hydra_zen/wrapper/_implementations.py

```diff
@@ -394,6 +405,32 @@ def hydra_main(
 
         kw = dict(config_name=config_name)
 
+        # For relative config paths, Hydra looks in the directory relative to the file
+        # in which the task function is defined. Unfortunately, it is only able to
+        # follow wrappers starting in Hydra 1.3.0. Thus `Zen.hydra_main` cannot
+        # handle string config_path entries until Hydra 1.3.0
+        if (config_path is _UNSPECIFIED_ and HYDRA_VERSION < Version(1, 2, 0)) or (
+            isinstance(config_path, str) and HYDRA_VERSION < Version(1, 3, 0)
+        ):  # pragma: no cover
+            warnings.warn(
+                "Specifying config_path via hydra_zen.zen(...).hydra_main "
+                "is only supported for Hydra 1.3.0+"
+            )
```

Comment:
The warning message would only seem to be relevant if `config_path` has been specified. Checking whether `config_path is _UNSPECIFIED_` in this `if` statement seems strange.

Reply:
I did this because if `version_base` is Hydra 1.1 and `config_path` is `_UNSPECIFIED_` then Hydra defaults `config_path` to be `"."`

But I realize that I need to have additional logic here to handle the case where the `version_base` via `zen` is specified